hera_cal/lstbin.py

```diff
@@ -62,6 +63,12 @@ def lst_bin(data_list, lst_list, flags_list=None, dlst=None, begin_lst=None, lst
     freq_array : type=ndarray, 1D array of unique data frequencies channels in Hz. Needed for rephase.
     lat : type=float, latitude of array in degrees North. Needed for rephase.
     verbose : type=bool, if True report feedback to stdout
+    nsamp_list : list of nsamples arrays
```

Comment:
This should appear after `flags_list` and should be called `nsamples_list` for consistency.

Reply:
I've changed the ordering.